src/main/scala/org/broadinstitute/dsde/firecloud/model/Trial.scala

```diff
@@ -98,4 +99,9 @@ object Trial {
 
   }
 
+  case class TrialProject(name: RawlsBillingProjectName, verified: Boolean, user: Option[WorkbenchUserInfo])
```

Comment:
I was hoping to avoid the `Option` and its corresponding requirements for `.get` and `.nonEmpty` etc ... the `TrialProject.apply` method defaults this to false.

But you're right - if we find we truly need to differentiate between "not verified" and "not checked" then we'd need to do something different. I might go with an enum in that case, with a few values like "unchecked", "creating", "verified", "failed", etc.